packages/fiori/src/WizardTab.js

```diff
@@ -113,16 +113,6 @@ const metadata = {
 			defaultValue: "listitem",
 		},
 
-		/**
-		 * Defines the aria-roledescription of the step.
-		 * @type {boolean}
-		 * @defaultvalue ""
-		 * @private
-		 */
-		ariaRoledescription: {
-			type: String,
-		},
-
 		/**
```

Comment:
I think that all of those aria attributes should be removed from the properties, they were used since the previous implementation gave the possibility to directly pass the aria attributes to the wizard step of custom element level